templates/startpage.html.twig

```diff
@@ -42,11 +42,102 @@
 
             <h2>What should we do?</h2>
             <p>
-                The short answer is: Whatever you want.
+                The short answer is: Whatever you want. If you want to have some
+                inspiration or ideas, see the following focus groups.
             </p>
 
+            <h4>Documentation</h4>
             <p>
-                This page will be updated shortly.
+                There is a lot of new features in Symfony 5.2 and some of them
+                have not been documented yet. See <a href="https://github.com/symfony/symfony-docs/milestone/23">the issues in the 5.2 milestone</a>
+                to find things to do. When start working with an issue, try to
+                see if the feature requires documentation or if the issue should
+                be closed. You may find the <a href="https://symfony.com/blog/category/living-on-the-edge">Symfony blog posts</a>
+                helpful.
+            </p>
+
+            <p>
+                There are also a few features introduced in Symfony 5.1 that are
+                missing documentation. See <a href="https://github.com/symfony/symfony-docs/milestone/23">the issues in the 5.1 milestone</a>.
+                These has been stalled because they don't need documentation or
+                because it may be complicated to write the documentation. For
+                each of these issues and PRs, please write a comment if you think
+                it should be closed or if they should be done. When a few people
+                given their opinion, it will be easier for a maintainer to take a
+                decision.
+            </p>
+
+            <p>
+                Feel free to create a PR for an issue if you want to.
+            </p>
+
+            <h4>Reproduce bugs</h4>
+            <p>
+                One thing that is super helpful for maintainers is reproducing bugs.
+                It is way easier to resolve a bug report with a code example and
+                that is verified by someone.
+            </p>
+
+            <p>
+                Have a look at the <a href="https://github.com/symfony/symfony/issues?q=is%3Aissue+is%3Aopen+label%3ABug">open bug reports</a>,
+                try to understand and reproduce the bugs. Make a comment like
+                "Verified" or "Cannot reproduce". Also explain what you have tried,
```

Comment:
```suggestion
                "Status: Reviewed" (if you can reproduce) or "Status: Works for me". Also explain what you have tried,
```

This way, Carson will correctly adjust the label.

Reply:
Maybe also link to https://symfony.com/doc/current/contributing/community/reviews.html#the-bug-report-review-process here

Reply:
Excellent idea